srcpkgs/gallery-dl/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'gallery-dl'
+pkgname=gallery-dl
+version=1.3.5
+revision=1
+noarch=yes
+build_style=python3-module
```

Comment:
add pycompile_modules="gallery_dl" under build_style=

Reply:
thank you. im not familiar with python and what the implications of this.